### DISCUSSION TITLE
Trying to get teacher benefits - GitHub only accepts student emails.

### DISCUSSION THREAD

**Discussion by GinnyBijnaar** [community/community#79133] [GitHub Education]
### Hi everyone,

Hi all, 
I am a lecturer and I want to setup a GitHub Classroom for one of my courses. When I apply for teacher benefits, the GitHub website only allows the student version of the emails of my institution (@student.uvs.edu), and not my staff email address (@uvs.edu).  I don't understand why I can't enter an email that is linked to the university website (www.uvs.edu). I don't want to use my personal email as it might cause my request to be denied. 

 It looks like this should be changed on GitHub's side, but it's impossible to find a human to report this to.

**Comment by MantasTech** [comment]
Hey Ginny,

To address this, you should contact GitHub Support directly.  You can create a support ticket using the GitHub Support portal.  Here is the link to contact [GitHub Support](https://docs.github.com/en/support/contacting-github-support).

I hope this helps in resolving the issue
